dev/archery/archery/cli.py

```diff
@@ -185,7 +224,7 @@ def benchmark(ctx):
 @click.option("--cmake-extras", type=str, multiple=True,
               help="Extra flags/options to pass to cmake invocation. "
               "Can be stacked")
-@click.argument("baseline", metavar="[<baseline>]]", default="master",
+@click.argument("baseline", metavar="[<baseline>]", default="WORKSPACE",
```

Comment:
What is this already? I can't find a reference for the git pseudo-revisions.

Reply:
it's not in git pseudo revisions, it's a magic keyword that only `archery` knows, it basically says, don't touch the git repository, use the workspace as-is (with dirty worktree). In `archery benchmark diff --help`:

> The special token "WORKSPACE" is reserved to specify the current git
  workspace. This imply that no clone will be performed.

Reply:
Hmm... if the baseline is the workspace, what is it compared against?

Reply:
That's the run sub-command, so it's not comparing against anything. Baseline might be mis-leading here, it probably should be revision?

Reply:
Yes, that would be better :)